.gitignore

```diff
@@ -13,5 +13,6 @@ build/*
 xcuserdata
 profile
 *.moved-aside
+PlaygroundUtility.remap
```

Comment:
I am not sure either. It appears randomly (my guess is the migrator), and carries some random lines of local changes to the playground sources that have been persisted already. Since deleting it does not affect the playground at all, I put it into `.gitignore`.